resources/dns/continuum-records.tf

```diff
@@ -0,0 +1,17 @@
+resource "cloudflare_record" "continuum_co" {
+  name    = "continuum.co"
+  proxied = false
+  ttl     = 3600
+  type    = "CNAME"
+  value   = "apex-loadbalancer.netlify.com"
+  zone_id = data.cloudflare_zone.continuum_co.id
+}
+
+resource "cloudflare_record" "continuum_co_www" {
+  name    = "www"
+  proxied = false
+  ttl     = 3600
+  type    = "CNAME"
+  value   = "continuum-collective.netlify.app"
```

Comment:
I wonder why this one is different

Reply:
One is for the apex domain and one is for subdomains, i guess they route each to different proxies.